README.md

```diff
@@ -1,24 +1,20 @@
+# Vonage Video API Network Test
```

Comment:
The README still has references to `opentok-network-test-js`. Please replace these with the new package name. 

And change `https://static.opentok.com/v2/js/opentok.js` to `https://unpkg.com/@vonage/client-sdk-video@2/dist/js/opentok.js`. And in the sample app, change `https://static.opentok.com/v2/js/opentok.min.js` to `https://unpkg.com/@vonage/client-sdk-video@2.29.2/dist/js/opentok.min.js`.

Also, in the readme and elsewhere (in the sample app), we should change variables with names like `otNetworkTest` to `videoNetworkTest` ... or something like that.

Reply:
And the comments in the src/NetworkTest/index.ts file still have references to `opentok-network-test-js`. Please update these.